.makim.yaml

```diff
@@ -504,4 +504,4 @@ groups:
         run: |
           # it requires the password manually
           ssh-keygen -R "[localhost]:2222" || true
-          ssh -o StrictHostKeyChecking=no testuser@localhost -p 2222 'pwd'
+          ssh -o StrictHostKeyChecking=no testuser@localhost -p 2222 'pwd'
```

Comment:
this change is not necessary, as you can see here, this red icon, means that you removed the empty line at the end of the file. you can see more information about that here: https://dev.to/documendous/why-you-should-end-your-source-files-with-a-new-line-156g

you can configure your IDE to do it automatically to you